### DISCUSSION TITLE
Does AvaloniaUI have a good method for calling a printer to print tables?

### DISCUSSION THREAD

**Discussion by Hub-moon** [AvaloniaUI/Avalonia#13944] [General]
Does AvaloniaUI have a good method for calling a printer to print tables?

**Comment by thevortexcloud** [comment]
Sort of but also not really. .NET in general does not have a lot of printing libraries and most of them are not cross platform. But I believe you can export the UI to a image/PDF and then print that. But you are better off using another library like [QuestPDF](https://github.com/QuestPDF/QuestPDF) for this and then some other library/method to send the data to the printer. EG on Windows you can use Adobe Acrobat with a launch argument to print a PDF.
